whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapConformance.java

```diff
@@ -0,0 +1,15 @@
+package net.ripe.db.whois.api.rdap;
+
+public enum RdapConformance {
+    CIDR_0("cidr0"), LEVEL_0("rdap_level_0"), PROFILE_0("nro_rdap_profile_0"), FLAT_MODEL("nro_rdap_profile_asn_flat_0");
+
```

Comment:
nro_rdap_profile_asn_flat_0 is only for Autnum objects. It should not be added in commons Can you confirm?